src/main/kotlin/nl/avisi/structurizr/site/generatr/site/views/SoftwareSystemDecisionsPage.kt

```diff
@@ -1,13 +1,35 @@
 package nl.avisi.structurizr.site.generatr.site.views
 
 import kotlinx.html.HTML
+import kotlinx.html.div
+import kotlinx.html.li
+import kotlinx.html.ul
 import nl.avisi.structurizr.site.generatr.site.model.SoftwareSystemDecisionsPageViewModel
 
 fun HTML.softwareSystemDecisionsPage(viewModel: SoftwareSystemDecisionsPageViewModel) {
-    if (viewModel.visible)
+    if (viewModel.visibleOnlyContainersDecisions) {
+        redirectRelative(
+                viewModel.decisionTabs.filter { it.visible }.first().link.relativeHref
```

Comment:
This can be simplified:

```suggestion
            viewModel.decisionTabs.first { it.visible }.link.relativeHref
```

Reply:
@dirkgroot that is cool! Didn't know this kotlin API.